pkg/asset/manifests/dns.go

```diff
@@ -230,13 +230,29 @@ func GCPNetworkName(project, network string) string {
 	return fmt.Sprintf("https://www.googleapis.com/compute/v1/projects/%s/global/networks/%s", project, network)
 }
 
+// GetProjectForDNSZones gets the project that should be contain the dns zones and dns records.
+func GetProjectForDNSZones(installConfig *installconfig.InstallConfig) string {
```

Comment:
Can this method return both project and zone? The caller can ignore the value they are not interested in.